maquetado/scss/styles.scss

```diff
@@ -157,3 +191,54 @@
     left: 0px;
   }
 }
+
+.login-img {
+  height: 70px;
+  margin: 0 24px;
+  width: 210px;
+}
+
+.login-detail {
+  background-color: $earls-green;
+  border-radius: 5px;
+  height: 30px;
+  width: 250px;
+  padding: 0 90px;
+  position: relative;
+  &::before {
+    content: '';
+    border-bottom: 2px solid $gray;
+    bottom: 0;
+    left: 0px;
+    margin-bottom: -20px;
+    position: absolute;
+    width: 100%;
+  }
+}
+
+.login-detail-font {
+  background-color: $earls-green;
+  color: $white;
+  font-size: 16px;
+  line-height: 20px;
+  margin: 5px;
+  width: 50px;
+}
+
+.sign-up-detail {
+  background-color: $wild-sand;
+  border: 2px solid $earls-green;
+  border-radius: 5px;
+  box-sizing: border-box;
+  height: 30px;
+  margin-top: 20px;
```

Comment:
margin-bottom in the previous element. Remember, try to avoid margin top and left